libs/platform/auth/login/login.component.ts

```diff
@@ -87,76 +95,65 @@ export class AuthLoginComponent extends ContentMixin<LoginOptions>(LitElement) {
       )}
 
       <form @submit=${this.onSubmit}>
-        <oryx-input
-          .label=${this.i18n('user.login.email')}
-          required
-          ?hasError="${this.hasError}"
-        >
-          <input
-            type="email"
-            name="email"
-            required
-            placeholder=${this.i18n('user.login.email')}
-          />
-        </oryx-input>
-
-        <oryx-password-input
-          .strategy="${this.$options()?.passwordVisibility}"
-          .label=${this.i18n('login.password')}
-          required
-          ?hasError="${this.hasError}"
-        >
-          <input
-            type="password"
-            name="password"
-            required
-            placeholder=${this.i18n('login.password')}
-          />
-        </oryx-password-input>
-
-        ${this.renderLoginOptions()}
+        ${this.fieldRenderer.buildForm(this.getFields())}
 
         <oryx-button .size=${ButtonSize.Md} ?loading=${this.isLoading}>
           <button slot="custom" ?disabled=${this.isLoading}>
             ${this.i18n('user.log-in')}
           </button>
         </oryx-button>
-      </form>`;
-  }
 
-  protected renderLoginOptions(): TemplateResult | void {
-    if (
-      !this.$options()?.forgotPasswordLink &&
-      !this.$options()?.enableRememberMe
-    ) {
-      return;
-    }
-    return html`
-      <div class="options">
-        ${when(
-          this.$options()?.enableRememberMe,
-          () => html`<oryx-checkbox>
-            <input
-              type="checkbox"
-              name="rememberme"
-              aria-label=${this.i18n('user.login.remember-me')}
-              ${this.rememberme}
-            />
-            ${this.i18n('user.login.remember-me')}
-          </oryx-checkbox>`
-        )}
         ${when(
           this.$options()?.forgotPasswordLink,
-          () => html`<oryx-link>
-            <a href=${this.$options()?.forgotPasswordLink}
-              >${this.i18n('user.login.forgot-password?')}</a
-            >
-          </oryx-link>`
+          () => html`
```

Comment:
This part is very broken. We should probably go with an oryx-link here but let's double check the designs.

Reply:
Yes, it should be oryx-link according to designs. Changed the oryx-button to oryx-link